TestAssets/CrossPublishTestProjects/StandaloneAppCrossPublish/project.json

```diff
@@ -0,0 +1,35 @@
+{
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "imports": [
+        "dnxcore50",
+        "portable-net45+win8"
+      ],
+      "dependencies": {
```

Comment:
I think the more standard is to put dependencies at the top level, isn't it?  At least for me it makes it more readable, since this only targets 1 TFM.